src/campaign/components/ItemModal/TagInput.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Input,
+  Tag,
+  TagCloseButton,
+  TagLabel,
+  TagProps,
+  Wrap,
+} from "@chakra-ui/react";
+import { KeyboardEventHandler, useState } from "react";
+
+export type TagInputProps = {
+  // eslint-disable-next-line no-unused-vars
+  onTagsChanged?: (tags: string[]) => void;
```

Comment:
General comment - there must be a way to configure the linter to understand this isn't a problem. I've had this issue a few times.